pandas/core/arrays/boolean.py

```diff
@@ -404,3 +405,25 @@ def _accumulate(
             return IntegerArray(data.astype(int), mask)._accumulate(
                 name, skipna=skipna, **kwargs
             )
+
+    def sum(
+        self,
+        *,
+        skipna: bool = True,
+        min_count: int = 0,
+        axis: AxisInt | None = 0,
+        **kwargs,
+    ):
+        arr = cast(BaseMaskedArray, self.astype("Int8"))
+        return arr.sum(skipna=skipna, min_count=min_count, axis=axis, **kwargs)
```

Comment:
this seems unrelated.  why is the parent class method not enough here?

Reply:
Yes, you are right, this isn't needed. I've pushed a new version.